usr/src/cmd/cmd-inet/usr.lib/mdnsd/DNSDigest.c

```diff
@@ -1,6 +1,7 @@
 /* -*- Mode: C; tab-width: 4 -*-
  *
  * Copyright (c) 2002-2011 Apple Inc. All rights reserved.
+ * Copyright (c) 2016 by Delphix. All rights reserved.
```

Comment:
No; not intentional. Thanks for noticing the mistake!

The original commit had fixed up whitespace issues which I dropped in the merge to openzfs/illumos, because the same fixes were already present in "master". I didn't realize there were no remaining changes to that file, so I'll remove this copyright.